gm_dotnet_managed/GmodNET/Startup.cs

```diff
@@ -36,7 +36,8 @@ internal static class Startup
 
                 if (native_version != full_assembly_version)
                 {
-                    throw new Exception("GmodNET version does not match with the version of dotnet_loader.");
+                    throw new Exception($"GmodNET version does not match with the version of dotnet_loader. Managed version: {full_assembly_version}; " +
```

Comment:
finally

Reply:
@SupinePandora43 😂 Some times even I'm not able to understand my exceptions.